site/public/js/docker_interface.js

```diff
@@ -64,6 +64,40 @@ function addFieldOnChange() {
     }
 }
 
+function confirmationdialog(url, id) {
```

Comment:
```suggestion
function confirmationDialog(url, id) {
```
Do this for better readability. Also make sure to fix where it's called.